builtin/logical/pki/path_config_crl.go

```diff
@@ -46,6 +48,12 @@ valid; defaults to 72 hours`,
 				Type:        framework.TypeBool,
 				Description: `If set to true, ocsp unauthorized responses will be returned.`,
 			},
+			"ocsp_expiry": {
+				Type: framework.TypeString,
```

Comment:
Smarter than I :-D I've always used `TypeDurationSecond` only for at the last minute. :-D

Reply:
This does make me wonder, can we safely convert this to a `TypeDurationSecond` field and upgrade the storage to use it? Everywhere we do these nasty `time.ParseDuration`, which could err, and it seems much nicer to just... Have the value? Just a thought :-)

Reply:
Hmm, the problem I have is we would lose what the end-user set through the config setting, which might make terraform style things very unhappy (setting 1h and getting back 3600 for example).

What would you think of having member vars stored in crlConfig that are the parsed values that return the duration value since I think we have a single reader now we could move that parsing into the cached copy?

Reply:
Ahh that's fair. Yeah, maybe an accessor that doesn't return an error?